test/output-validation.test.js

```diff
@@ -0,0 +1,121 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const request = require('request')
+const fastify = require('..')()
+
+const schema = {
+  response: {
+    '200': {
```

Comment:
Yes, if the key is a number I'm able to use it directly. But sadly, with the 2xx the ' is required